package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -415,6 +415,7 @@
 from six.moves import range, zip_longest
 
 import numpy as np
+import sys
```

Comment:
You should not need `sys` for anything. For output, use [`lib.log.ProgressMeter`](http://www.mdanalysis.org/mdanalysis/documentation_pages/lib/log.html?highlight=progressmeter#MDAnalysis.lib.log.ProgressMeter) (see below comment).